Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -268,9 +268,9 @@ function IsFeatureEnabled
 #
 function ExitOnPRBuild
 {    
-    if ((IsPrBuild) -and !(IsFeatureEnabled "SQPullRequestBot" $false))
+    if ((IsPrBuild) -and !(IsFeatureEnabled "SQPullRequestBot" $true))
```

Comment:
"SQPullRequestBot" is a build variable that users can set. It now defaults to true so if a user: 

- defines the variable and sets it to any value other than "false" OR
- does not define the variable 

then PRCA is enabled

If the variable is defined and set to "false" then PRCA will be disabled.